tests/typegen/strats/test_inline.py

```diff
@@ -35,47 +35,67 @@ def visit_FunctionDef(self, node: ast.FunctionDef) -> None:
                 for arg in node.args.args:
                     assert arg.annotation is None, f"{ast.dump(arg)}"
 
+        elif node.name == "__init__":
```

Comment:
Why do we skip this?

Reply:
Currently, there aren't any args to check. But a check for __init__ is necessary otherwise the program will move to the else block and assert false.

Reply:
Ah this is just test code anyway